xtask/src/commands/check.rs

```diff
@@ -1,14 +1,40 @@
-use anyhow::Result;
+use anyhow::{ensure, Result};
+use std::fs::read_to_string;
 use structopt::StructOpt;
 use xtask::*;
 
+static LICENSES_HTML_PATH: &str = "licenses.html";
+
 #[derive(Debug, StructOpt)]
 pub struct Compliance {}
 
 impl Compliance {
     pub fn run(&self) -> Result<()> {
         cargo!(["deny", "-L", "error", "check"]);
 
+        eprintln!("checking generated licenses.html file");
+
+        let licenses_html_before = read_to_string(LICENSES_HTML_PATH)?;
+
+        cargo!([
+            "about",
+            "-L",
+            "error",
+            "generate",
+            "--workspace",
+            "-o",
+            "licenses.html",
+            "about.hbs"
+        ]);
+
+        let licences_html_after = read_to_string(LICENSES_HTML_PATH)?;
```

Comment:
If the file gets too big you might want to hash it instead. But I'm ok with this for now.

Reply:
true, especially since I don't show any kind of diff. let's keep this in mind!

Reply:
Ok it's up to date now!